README.md

```diff
@@ -145,6 +145,7 @@ module "hive" {
 ```
 
 ### Verifying setup
+
```

Comment:
```suggestion
```

Reply:
yeah. I had to trigger new actions run, due I cannot cancel this run https://github.com/fredrikhgrelland/terraform-nomad-hive/actions/runs/321493269 . I tried to cancel, but it did not work

Reply:
Oh :sweat_smile: